apps/election-manager/src/config/types.ts

```diff
@@ -88,17 +88,17 @@ export interface ContestTallyMeta {
 export type ContestTallyMetaDictionary = Dictionary<ContestTallyMeta>
 
 export interface Tally {
-  readonly precinctId?: string
-  readonly scannerId?: string
   readonly numberOfBallotsCounted: number
-  readonly castVoteRecords: ReadonlyMap<string, CastVoteRecord>
+  // TODO(#2975): Once we're removing duplicate ballots, make this a dictionary indexed by ballotId
+  readonly castVoteRecords: CastVoteRecord[]
```

Comment:
```suggestion
  readonly castVoteRecords: readonly CastVoteRecord[]
```

Reply:
oh I assumed having the readonly before the variable name made this readonly, whats the difference between having it there and having it before the type declaration? Is one saying the items in the array are readonly in addition to the whole array or something?

Reply:
`readonly castVoteRecords: CastVoteRecord[]` means you can't do `tally.castVoteRecords = []`, whereas `readonly castVoteRecords: readonly CastVoteRecord[]` means you can't do `tally.castVoteRecords.push(…)`. The latter does not modify the identity of the property, which is what the first `readonly` protects.